examples/sample.py

```diff
@@ -36,12 +38,14 @@
                     default=False,
                     help='Indicates if training has to be resumed from the latest checkpoint')
 
-opt = parser.parse_args()
-print(opt)
+fileConfig('logging_config.ini',disable_existing_loggers=False)
```

Comment:
The path is relevant, it doesn't work when running sample.py from the root folder, as instructed in README.